pyServer/GuestFS.py

```diff
@@ -74,6 +74,8 @@ def callGF(self, echoStr, commands, continueOnError=False, returnRawResults=Fals
                 if err:
                     if continueOnError:
                         self.rootLogger.warning('GuestFish:' + echoStr + ':Error> \r\n' + err)
+                        self.rootLogger.warning('GuestFish: Retrying for' + echoStr)
+                        retValue = self.callGFRetry(echoStr, retArgs)
```

Comment:
Given the similarity between callGF and callGFRetry, can we just have a single function with an `isRetry` or `retryCount` variable that allows for a slightly different path to be taken on retries w/o function duplication?

Reply:
Applied changes as suggested - added retryCount and removed duplicate function.